modules/templates/DIRECT/menus.py

```diff
@@ -217,6 +219,8 @@ def inv():
         """ INV / Inventory """
 
         ADMIN = current.session.s3.system_roles.ADMIN
+        ORG_ADMIN = current.session.s3.system_roles.ORG_ADMIN
+        SUPPLY_COORDINATOR = current.session.s3.system_roles.SUPPLY_COORDINATOR
```

Comment:
SUPPLY_COORDINATOR is not a system role, but a template-specific "business role", so it shouldn't be looked up from system_roles.